app/views/balances/show.html.haml

```diff
@@ -0,0 +1,100 @@
+- if @balance
+  - if browser.device.mobile?
+    .mobile
+      %h1.text-center
+        Твій баланс
+      .balance
+        .content
+          .align-items
+            .row.text-center
+              .amount.col-sm-4.item
+                %strong= "#{@balance.account_balance.round}₴"
+                %br
+                Баланс
+      .col-md-12
+        .operations-list
+          %h1 Історія Балансу
+          .table
+            .container
+              .row.title.d-flex.align-items-center
+                .col-md-8
+                  %b  Subject
+            - operations = @balance.payments + @balance.replenishments + @balance.ready_billings
+            - operations.sort_by(&:created_at).reverse.each do |o|
```

Comment:
extract logic and name it (provide human friendly definitions e.g. recent_operations or operations_history